spec/omniauth/strategies/drip_spec.rb

```diff
@@ -26,4 +26,13 @@
     end
   end
 
+  context 'uid' do
```

Comment:
This spec came from https://github.com/calebjclark/omniauth-constantcontact/blob/master/spec/omniauth/strategies/constantcontact_spec.rb#L30